.github/workflows/ci.yml

```diff
@@ -20,6 +20,7 @@ jobs:
           node-version: ${{ matrix.node }}
       - run: npm install
       - run: npm test
+      - run: npm test-esm
```

Comment:
```suggestion
      - run: npm run test-esm
```

Reply:
ar right. I'm using `pnpm` so forget `npm` needs that. :)